app/javascript/dashboard/routes/dashboard/conversation/contact/ContactInfoRow.vue

```diff
@@ -2,11 +2,14 @@
   <div class="contact-info--row">
     <a v-if="href" :href="href" class="contact-info--details">
       <i :class="icon" class="contact-info--icon" />
-      <span v-if="value" class="text-truncate">{{ value }}</span>
+      <span v-if="value" class="text-truncate" :title="value">{{ value }}</span>
       <span v-else class="text-muted">{{
         $t('CONTACT_PANEL.NOT_AVAILABLE')
       }}</span>
+
+      <span v-if="showCopy" class="copy-icon ion-clipboard" @click="onCopy" />
```

Comment:
span is not an interactive element. It would be good if you can make it a button with a span.